src/groups/mqb/mqbblp/mqbblp_remotequeue.cpp

```diff
@@ -357,10 +357,12 @@ void RemoteQueue::pushMessage(
 {
     // executed by the *QUEUE DISPATCHER* thread
 
-    mqbi::StorageMessageAttributes attributes(0ULL,  // Timestamp; unused
-                                              1,     // RefCount
-                                              messagePropertiesInfo,
-                                              compressionAlgorithmType);
+    mqbi::StorageMessageAttributes attributes(
+        0ULL,  // Timestamp; unused
+        1,     // RefCount
+        0,     // appData length (cannot deduce it now: appData might be 0)
+        messagePropertiesInfo,
+        compressionAlgorithmType);
```

Comment:
Had a bug in this initialization before.
Called `appData->length()` to initialize `attributes` on the spot, but `appData` might be null pointer in this context, and this caused crash